src/Compilers/Test/Core/MarkedSource/SourceWithMarkedNodes.cs

```diff
@@ -8,41 +8,48 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Linq;
-using System.Text;
 using System.Text.RegularExpressions;
-using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Text;
-using Roslyn.Utilities;
 using Xunit;
 
 namespace Roslyn.Test.Utilities
 {
-    internal sealed class SourceWithMarkedNodes
+    internal sealed partial class SourceWithMarkedNodes
     {
         public readonly string Source;
+        public readonly string StrippedSource;
+        public readonly string Input;
         public readonly SyntaxTree Tree;
```

Comment:
>public readonly SyntaxTree Tree; [](start = 8, length = 32)

It's a bit odd that we have `Tree` that matches the `Source` but not `StrippedSource`.